mesonbuild/interpreter.py

```diff
@@ -874,7 +875,28 @@ def get_define_method(self, args, kwargs):
         extra_args = self.determine_args(kwargs)
         deps = self.determine_dependencies(kwargs)
         value = self.compiler.get_define(element, prefix, self.environment, extra_args, deps)
-        mlog.log('Checking for value of define "%s": %s' % (element, value))
+        mlog.log('Fetching value of define "%s": %s' % (element, value))
+        return value
+
+    def get_return_value_method(self, args, kwargs):
```

Comment:
What else would you call the value returned by a function though? Also, the return value from main in an executable is called an "exit code" or "exit status", isn't it? See: https://www.gnu.org/software/libc/manual/html_node/Exit-Status.html